src/crimson/osd/osd_operations/client_request.cc

```diff
@@ -204,12 +201,11 @@ auto ClientRequest::reply_op_error(const Ref<PG>& pg, int err)
   reply->set_reply_versions(eversion_t(), 0);
   reply->set_op_returns(std::vector<pg_log_op_return_item_t>{});
   return conn->send(std::move(reply)).then([] {
-    return seastar::make_ready_future<ClientRequest::seq_mode_t>
-      (seq_mode_t::OUT_OF_ORDER);
+    return seastar::make_ready_future<>();
```

Comment:
could just drop this trailing continuation.

Reply:
https://github.com/ceph/ceph/pull/49127/files#diff-3465c956b7a8e8fab614e23efef1c6093d8277c66ac36a78a138040010a92eceR203